src/node_perf.cc

```diff
@@ -58,20 +54,9 @@ void performance_state::Mark(enum PerformanceMilestone milestone,
 }
 
 double GetCurrentTimeInMicroseconds() {
-#ifdef _WIN32
-// The difference between the Unix Epoch and the Windows Epoch in 100-ns ticks.
-#define TICKS_TO_UNIX_EPOCH 116444736000000000LL
-  FILETIME ft;
-  GetSystemTimeAsFileTime(&ft);
-  uint64_t filetime_int = static_cast<uint64_t>(ft.dwHighDateTime) << 32 |
-                          ft.dwLowDateTime;
-  // FILETIME is measured in terms of 100 ns. Convert that to 1 us (1000 ns).
-  return (filetime_int - TICKS_TO_UNIX_EPOCH) / 10.;
-#else
-  struct timeval tp;
-  gettimeofday(&tp, nullptr);
-  return MICROS_PER_SEC * tp.tv_sec + tp.tv_usec;
-#endif
+  uv_timeval_t tv;
+  CHECK_EQ(uv_gettimeofday(&tv), 0);
+  return MICROS_PER_SEC * tv.tv_sec + tv.tv_usec;
```

Comment:
`tv_sec` and `tv_usec` are of type `long` and so might be 32 bits. The calculation works okay because `MICROS_PER_SEC` is of type `double` and therefore implicitly promotes the other two to that type but that's not very obvious.

Maybe add an explicit `static_cast<double>(MICROS_PER_SEC)` or replace it with an inline `constexpr double kMicrosecondsPerSecond = 1e6`?